src/jit/lsra.h

```diff
@@ -538,6 +538,22 @@ class LinearScan : public LinearScanInterface
     LsraSpill                   getLsraSpill()                  { return (LsraSpill) (lsraStressMask & LSRA_SPILL_MASK); }
     bool                        spillAlways()                   { return getLsraSpill() == LSRA_SPILL_ALWAYS; }
 
+    // This controls whether RefPositions that require a register optionally should
+    // be allocated a busy register by spilling another less important RefPosition.
+    enum LsraRegOptionalControl     { LSRA_REG_OPTIONAL_DEFAULT          = 0,
+                                      LSRA_REG_OPTIONAL_NO_BUSYREG_ALLOC = 0x1000,
+                                      LSRA_REG_OPTIONAL_MASK             = 0x1000 };
+
+    LsraRegOptionalControl      getLsraRegOptionalControl()            
+    { 
+        return (LsraRegOptionalControl) (lsraStressMask & LSRA_REG_OPTIONAL_MASK); 
+    }
+
+    bool                        regOptionalNoBusyRegAlloc()     
+    { 
+        return getLsraRegOptionalControl() == LSRA_REG_OPTIONAL_NO_BUSYREG_ALLOC;  
+    }
+
```

Comment:
The reason i thought would be useful is that it would change the spill pattern.  Current allocation mechanism for a reg optional ref position is

```
Try to allocate a free reg
if no free reg available check to see if we can spill a less important interval to get a register.
```

This stress mode would change that to

```
Try to allocate a free reg
if one not available, don't allocate since it is marked reg optional.
```